app/views/foreman_ansible/ansible_roles/_ansible_row_buttons.html.erb

```diff
@@ -0,0 +1,15 @@
+<td>
+    <%=
+      if hostgroup.all_ansible_roles.empty?
```

Comment:
Can you add the button directly to the list of actions of the hostgroup? http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin contains some help